src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/mha_single_token.cpp

```diff
@@ -189,7 +189,96 @@ static void attn_acc_value_block(float* out, float* weight, T* v, size_t S, size
 }
 
 static void attn_acc_value_block(float* out, float* weight, uint8_t* v, size_t S, size_t block_size) {
```

Comment:
Does it really needs to be fully separate function from what you have implemented for SDPA? No reuse possible at all?

Reply:
Yes, it does. In PR https://github.com/openvinotoolkit/openvino/pull/24107 the SDPA and PagedAttention will be split into different files.
A new no reuse function because:
1, kv_cache of PA is blocked: [16, S] instead of [1, S] in SDPA indirect kvcache, this brings new optimization opportunities: share reading Q or (Q*K') for serveral tokens and unrolling in tokens direction instead of S which can save 3*_mm512_add_ps per token
2, u8 kv_cache has totally different layout to match vllm, this requires the function parameters are different.